test/f90_correct/vector.f90

```diff
@@ -0,0 +1,21 @@
+! RUN: %flang -S -emit-llvm %s -o - | FileCheck %s -check-prefix=METADATA
+! RUN: %flang -Hx,59,2 -S -emit-llvm %s -o - | FileCheck %s -check-prefix=ENABLE-METADATA
```

Comment:
This FileCheck prefixes for the negative tests are confusing.

Reply:
Done.